packages/sdk/src/webhooks/client.ts

```diff
@@ -92,6 +111,107 @@ export class LinearWebhookClient {
     return handler as LinearWebhookHandler;
   }
 
+  /**
+   * Determines whether the provided value is a Fetch API `Request`.
+   * Used as a type guard to select the appropriate runtime path.
+   *
+   * @param value - Unknown request-like value
+   * @returns True if `value` is a Fetch API `Request`
+   */
+  private isFetchRequest(value: unknown): value is Request {
+    const candidate = value as { arrayBuffer?: unknown } | null | undefined;
+    return typeof candidate?.arrayBuffer === "function";
```

Comment:
nit: rather than casting, what about:

```ts
    return (
      typeof value === "object" && value !== null && "arrayBuffer" in value && typeof value.arrayBuffer === "function"
    );
```